mon-pix/app/services/base-url.js

```diff
@@ -0,0 +1,131 @@
+// This file is a copy of an original file from mon-pix.
+// If you need a change, modify the original file and
+// propagate the changes in the copies in all the fronts.
+
+import Service, { service } from '@ember/service';
+import ENV from 'mon-pix/config/environment';
+
+const { DEFAULT_LOCALE } = ENV.APP;
+
+// Don't use directly this service, use a url service which extends it
+export default class BaseUrlService extends Service {
+  @service currentDomain;
+  @service locale;
+
+  pixAppUrlWithoutExtension = ENV.APP.PIX_APP_URL_WITHOUT_EXTENSION;
+
+  get homeUrl() {
+    const currentLanguage = this.locale.currentLocale;
+    return `${ENV.rootURL}?lang=${currentLanguage}`;
+  }
+
+  get serverStatusUrl() {
+    const currentLanguage = this.locale.currentLocale;
+    return `https://status.pix.org/?locale=${currentLanguage}`;
+  }
+
+  get pixAppCampaignRootUrl() {
+    if (!this.pixAppUrlWithoutExtension) return null;
+    const tld = this.currentDomain.getExtension();
+    return `${this.pixAppUrlWithoutExtension}${tld}/campagne`;
+  }
+
+  get pixAppForgottenPasswordUrl() {
+    if (!this.pixAppUrlWithoutExtension) return null;
+    const tld = this.currentDomain.getExtension();
+    const url = `${this.pixAppUrlWithoutExtension}${tld}/mot-de-passe-oublie`;
+    const locale = this.locale.currentLocale;
+    return locale === 'fr' ? url : `${url}?lang=${locale}`;
```

Comment:
Pour être cohérent avec les autres fonctions : 
```suggestion
    const currentLocale = this.locale.currentLocale;
    return currentLocale === 'fr' ? url : `${url}?lang=${currentLocale}`;
```